### PR TITLE
Pass IPAM network CIDR as parameter to the operator

### PR DESCRIPTION
**Pull request by fiunchinho**
I used the CIDR in the installations repository as default.

### CODE REVIEW

File: helm/azure-operator/templates/configmap.yaml [hunk 1]
```diff
@@ -27,6 +27,11 @@ data:
       installation:
         name: '{{ .Values.Installation.V1.Name }}'
       {{- if .Values.Installation.V1.Guest }}
```
Comment (tuommaki) [inline helm/azure-operator/templates/configmap.yaml:29]: Not part of this PR, but what's this if? Naming suggest like this is adaptation for CP and TC app, but it doesn't make sense for `azure-operator`. Something else?

Comment (fiunchinho) [reply helm/azure-operator/templates/configmap.yaml:29]: Not sure. Maybe it was to check if OIDC parameters were being passed?

Comment (tuommaki) [reply helm/azure-operator/templates/configmap.yaml:29]: :man_shrugging:
